cns/NetworkContainerContract.go

```diff
@@ -107,16 +107,130 @@ type ConfigureContainerNetworkingRequest struct {
 	NetworkContainerid string
 }
 
-// KubernetesPodInfo is an OrchestratorContext that holds PodName and PodNamespace.
-type KubernetesPodInfo struct {
```

Comment:
CNS is used in other orchestrators as well like ACI, So not sure if we should rename the struct name.


For some scenarios, Pod Key is still name + namespace

Reply:
I'm worried for using the new struct in other scenarios. can we create a separate struct for AKS scenario, then later converge to one.